app/Http/Controllers/Administration/AdministrationExportController.php

```
@@ -0,0 +1,53 @@
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Storage;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+class AdministrationExportController extends Controller
+{
+    public function index(): \Illuminate\View\View
+    {
+        return view('administration.export.index');
+    }
+
+    public function create(Request $request): RedirectResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)